### ISSUE TITLE
Ubuntu 16 build caveats

### ISSUE THREAD

**Issue by rocket-pig** [Antox/Antox#396]
A couple of things that might save others hours of chasing things down:

if you store the project files in an encrypted home directory, builds will fail with "...cannot extract zip".
Solution is to move project files to /opt (or somewhere else) before attempting to compile.

Attempting to download/compile in any other fashion than this causes all sorts of weirdness. Using Android Studio's built-in git support etc will Not work. 

git clone https://github.com/Antox/Antox
cd Antox/
./download-dependencies.sh
./gradlew build -x lint --parallel --debug

update, I can compile/run in Android Studio after this by choosing "Import Gradle Project" instead of file>open.

**Comment by zoff99** [comment]
did you try it with the development version v0.25 aswell? does that compile?
https://github.com/Antox/Antox/pull/392

```
git clone https://github.com/zoff99/Antox
cd Antox/
git checkout "zoff99/Antox_v0.25"
./download-dependencies.sh
./gradlew build -x lint --parallel --debug
```

**Comment by rocket-pig** [comment]
is "development build" a bit misleading? it appears youre a separate developer with a pull request? im very new user to github so I can't quite make out who's developing what. 

im wanting to re-theme or add a dark/invert theme option to settings, that's all really, and learn github / android studio basic practices. I'm not sure how compiling your unmerged branch will help me? forgive me if im not understanding the github structure very well.

**Comment by zoff99** [comment]
the branch is in discussion with @wiiaam 
also i am trying to find out if your compile issue is fixed in 0.25 or not

**Comment by zoff99** [comment]
is this still broken with the latest master?
